app/lib/frontend/handlers/account.dart

```diff
@@ -134,66 +127,6 @@ Future<ClientSessionStatus> getAccountSessionHandler(
   );
 }
 
-/// Handles POST /api/account/session
-Future<shelf.Response> updateSessionHandler(
-    shelf.Request request, ClientSessionRequest body) async {
-  final sw = Stopwatch()..start();
-  final authenticatedUser = await requireAuthenticatedWebUser();
-  final user = authenticatedUser.user;
-
-  InvalidInputException.checkNotNull(body.accessToken, 'accessToken');
-  await accountBackend.verifyAccessTokenOwnership(body.accessToken!, user);
-  final t1 = sw.elapsed;
-
-  // Only allow creation of sessions on the primary site host.
-  // Exposing session on other domains is a security concern.
-  // Note: staging sites may have a different primary host.
-  if (request.requestedUri.host != activeConfiguration.primarySiteUri.host ||
-      request.requestedUri.scheme !=
-          activeConfiguration.primarySiteUri.scheme) {
-    // The resource simply doesn't exist on this host.
-    throw NotFoundException.resource('no such url');
-  }
-
-  final cookies = parseCookieHeader(request.headers[HttpHeaders.cookieHeader]);
-  final sessionData =
-      await accountBackend.parseAndLookupUserSessionCookie(cookies);
-  final t2 = sw.elapsed;
-  // check if the session data is the same
-  if (sessionData != null &&
-      sessionData.userId == user.userId &&
-      sessionData.email == user.email) {
-    final status = ClientSessionStatus(
-      changed: false,
-      expires: sessionData.expires,
-      authenticatedAt: null,
-    );
-    _logger.info(
-        '[pub-session-handler-debug] Session was alive: $t1 / ${t2 - t1}.');
-    return jsonResponse(status.toJson());
-  }
-
-  final profile = await authProvider.getAccountProfile(body.accessToken);
-  final t3 = sw.elapsed;
-  final newSession = await accountBackend.createNewUserSession(
-    name: profile!.name!,
-    imageUrl: profile.imageUrl!,
-  );
-  final t4 = sw.elapsed;
-  _logger.info(
-      '[pub-session-handler-debug] Session was created: $t1 / ${t2 - t1} / ${t3 - t2} / ${t4 - t3}.');
-
-  return jsonResponse(
-    ClientSessionStatus(
-      changed: true,
-      expires: newSession.expires,
-      authenticatedAt: null,
-    ).toJson(),
-    headers: session_cookie.createUserSessionCookie(
-        newSession.sessionId, newSession.expires),
-  );
-}
-
 /// Handles DELETE /api/account/session
```

Comment:
So in the future, it'd make a lot of sense if we move this to `/sign-out`

Reply:
Maybe, however I don't see much benefit to gain there. Also, that could be an annoying attack if somebody sends you to the /sign-out link and now the session is signed out.